Samples/Samples/Converters/TimeSelectedEventArgsConverter.cs

```diff
@@ -13,8 +13,8 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
         {
             var eventArgs = value as PropertyChangedEventArgs;
 
-            if (eventArgs == null || eventArgs.PropertyName != TimePicker.TimeProperty.PropertyName)
-                return value;
+            if (eventArgs == null || eventArgs.PropertyName != nameof(TimePicker.TimeProperty))
```

Comment:
For the event we're interested is the PropertyName set to "Time" or "TimeProperty"?

Reply:
Yep Time, instead of TimeProperty. TimeProperty seems to work in android but not iOS.

Have altered in next PR